app/gui2/src/components/lexical/sync.ts

```diff
@@ -32,13 +32,15 @@ export function useLexicalSync<T>(
     state.value = editorState
   })
 
-  const getContent = computed(() => {
+  const getEditedContent = computed(() => {
     return state.value.read($read)
   })
 
+  const getContent = () => editor.getEditorState().read($read)
+
   return {
     content: {
-      state: getContent,
+      state: getEditedContent,
```

Comment:
`content.state` is still misleading, because it is not the actual _state_ of the editor, but rather a one-way view of user edits.